### PR TITLE
docs(schema): ensure Advanced pages are under Advanced section

### PR DESCRIPTION
**Pull request by hassankhan**
Story details: https://app.clubhouse.io/cube-dev/story/758

### CODE REVIEW

File: docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+permalink: /schema/advanced/export-import
+category: Data Schema
+subCategory: Advanced
+menuOrder: 1
+redirect_from:
+  - /export-import
+  - /recipes/export-import
+---
+
+In Cube.js, your data schema is code, and code is much easier to manage when it
```
Comment (rpaik) [inline docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md:12]:
```suggestion
In Cube.js, your data schema is the code, and code is much easier to manage when it
```

File: docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+is in small, digestible chunks. It is best practice to keep files small and
+containing only relevant and not duplicated code. As your data schema grows,
```
Comment (rpaik) [inline docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md:14]:
```suggestion
containing only relevant and nonduplicated code. As your data schema grows,
```

**Comment by rpaik** [comment]
Just a couple of suggestions @hassankhan 😃

**Review by adnanrahic — APPROVED** [review]
LGTM :ok_hand:

**Pull request merged**